barretenberg/cpp/src/barretenberg/commitment_schemes/ipa/ipa.hpp

```diff
@@ -9,13 +9,63 @@
 #include <string>
 #include <vector>
 
+namespace bb {
+
 /**
- * @brief IPA (inner-product argument) commitment scheme class. Conforms to the specification
- * https://hackmd.io/q-A8y6aITWyWJrvsGGMWNA?view.
+ * @brief IPA (inner product argument) commitment scheme class.
```

Comment:
Looks like this:
![image](https://github.com/AztecProtocol/aztec-packages/assets/4798775/51a8054b-0f66-4682-870e-ddfea446e90b)